pmd-plsql/src/test/resources/net/sourceforge/pmd/lang/plsql/ast/IsOfType.pls

```diff
@@ -17,7 +17,7 @@ IF ioFPOobj IS NOT OF TYPE (ONLY FPOGE_OBJ) THEN
 loFPOGE_OBJ:=treat(ioFPOobj AS FPOGE_OBJ);
 end if;
 
-loFPOGE_OBJ:=SELECT A FROM persons p WHERE VALUE(p) IS OF TYPE (employee_t);
-loFPOGE_OBJ:=SELECT A FROM persons p WHERE VALUE(p) IS NOT OF TYPE (ONLY employee_t, other_t);
+SELECT A into lo_FPOGE_OBJ FROM persons p WHERE VALUE(p) IS OF TYPE (employee_t);
```

Comment:
That might be the result of the commented out "SelectStatement()" above in the grammar.

Before, the Select was part of a expression, now it is a select statement on its own. I think, we need both variants.

Reply:
No, this is just invalid. IOW this will result in a parsing error in SQL*Plus as well. So I think that PMD must not accept this.
It is not even valid PL/SQL AFAIK .

Reply:
Ah, ok. Yes, in that case, PMD must not accept this as well. Thanks!